### PR TITLE
Remove the Extractable contract and feature

### PR DESCRIPTION
**Pull request by nuevoalex**

### CODE REVIEW

File: source/contracts/trading/ICash.sol [hunk 1]
```diff
@@ -9,5 +9,4 @@ contract ICash is ERC20 {
     function depositEtherFor(address _to) external payable returns(bool);
     function withdrawEther(uint256 _amount) external returns(bool);
     function withdrawEtherTo(address _to, uint256 _amount) external returns(bool);
-    function getTypeName() public view returns (bytes32);
```
Comment (adrake33) [inline source/contracts/trading/ICash.sol]: Why was this removed? Is it no longer needed?

Comment (nuevoalex) [reply source/contracts/trading/ICash.sol]: Cash already inherits from this so it was redundant
